webapp/pages/profile/_id/_slug.vue

```diff
@@ -361,6 +360,12 @@ export default {
         fetchPolicy: 'cache-and-network',
       })
     },
+    setActivePosts() {
+      if (!this.Post) {
+        return []
+      }
+      return this.uniq(this.Post.filter(post => !post.deleted))
+    },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-14T11:04:40Z" title="Friday, June 14th 2019, 1:04:40 pm +02:00">Jun 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -365,6 +362,12 @@ export default {
         fetchPolicy: 'cache-and-network',
       })
     },
+    setActivePosts() {
+      if (!this.Post) {
+        return []
+      }
+      return this.uniq(this.Post.filter(post => !post.deleted))
+    },
```


could you explain what motivated these changes @Tirokk 
sorry, it's not obvious to me

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-06-14T11:24:29Z" title="Friday, June 14th 2019, 1:24:29 pm +02:00">Jun 14, 2019</time>_

---


This code is probably written by @roschaefer .

Here I rewrote it a little to get the delete Post on the User Profile working again. But nothing serious I think.
He filters the deleted Posts out of the list array I think.